test_ws_client.py

```
@@ -0,0 +1,256 @@
+                    await receiver
+                except asyncio.CancelledError:
+                    pass
+            else:
+                # Просто слухаємо події
+                print("Listening for events... (Press Ctrl+C to stop)")
+                print("-" * 60)
+                await receive_messages(websocket)
+
+    except KeyboardInterrupt:
+        print("\n\nDisconnected by user")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)